tests/dummy/app/pods/company-custom/model.js

```diff
@@ -0,0 +1,21 @@
+export default {
+  type: 'object',
+  properties: {
+    username: {
+      type: 'string'
+    },
+    password: {
+      type: 'string'
+    },
+    rememberMe: {
```

Comment:
the remember box? it does correctly have a true/false value but I'm not sure how it will tie in to actually remember the user info.